src/ADAL.PCL/PlatformInformationBase.cs

```diff
@@ -45,6 +45,22 @@ internal abstract class PlatformInformationBase
 
         public abstract string GetDeviceModel();
 
+        public abstract string GetDeviceId();
+
+        public abstract string GetApplicationName();
+
+        public abstract string GetApplicationVersion();
+
+        public virtual string GetBrokerAppName()
+        {
+            return "Authenticator App";
+        }
+
+        public virtual string GetBrokerVersion()
+        {
+            return "2";
```

Comment:
This is the maximum version that ADAL talks to and is hard coded as "2".Hence I used the same as default value to provide the best estimate.